src/databases/functions/users/get-user-by-user-id.js

```diff
@@ -1,12 +1,14 @@
-const { User } = require("../../models")
+const {User} = require('../../models');
 
 module.exports = async (userId) => {
-    if(userId == null) return;
-    let user = await User.findOne({
-        where: {
-            user_id: userId
-        }
-    })
-    if(user == null) return []
-    return user
-}
+  if (userId == null) return;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for `userId` being null is loose, it might lead to unexpected behavior if `userId` is `0` or `""`. It's better to use strict equality (`===`) for comparison.
```diff
-  if (userId == null) return;
+  if (userId === null || userId === undefined) return;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->